### PR TITLE
HotFix: Fix the typo and keep code-server case-sensitive

### PR DESCRIPTION
**Pull request by harshad16**
HotFix: Fix the typo and keep code-server case-sensitive
Refer: https://redhat-internal.slack.com/archives/C05TTTYG599/p1706184360108179?thread_ts=1706112551.862629&cid=C05TTTYG599

### CODE REVIEW

File: manifests/base/code-server-notebook-imagestream.yaml [hunk 1]
```diff
@@ -6,8 +6,8 @@ metadata:
     opendatahub.io/notebook-image: "true"
   annotations:
     opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/tree/main/codeserver"
-    opendatahub.io/notebook-image-name: "Code server"
-    opendatahub.io/notebook-image-desc: "Code server workbench allows users to code, build, and collaborate on projects directly from web."
+    opendatahub.io/notebook-image-name: "code-server"
+    opendatahub.io/notebook-image-desc: "code-server workbench allows users to code, build, and collaborate on projects directly from web."
```
Comment (jiridanek) [inline manifests/base/code-server-notebook-imagestream.yaml:10]: @jstourac collaborate? Do you know what feature this refers to specifically? Can more than one person access the code-server at the same time and use the web ui?

File: manifests/base/code-server-notebook-imagestream.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ spec:
     # N Version of the image
     - annotations:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
-        opendatahub.io/notebook-python-dependencies: '[{"name":"Code-sever","version":"4.16"}]'
+        opendatahub.io/notebook-python-dependencies: '[{"name":"code-server","version":"4.16"}]'
```
Comment (jiridanek) [inline manifests/base/code-server-notebook-imagestream.yaml:20]: +1, looks better than Code-server to me

**Review by jiridanek — APPROVED** [review]

File: manifests/base/code-server-notebook-imagestream.yaml (hunk 1 above)
Comment (harshad16) [reply manifests/base/code-server-notebook-imagestream.yaml:10]: all the workbenches can be accessed at the same time if the owner of the workbench provides access to the user.
example:
if multiple people have access to a DS project.
they all can use all the workbenches spined up in the DS project.

**Comment by harshad16** [comment]
As both QA and Dev team have signed off
Merging this in.

**Pull request merged**